pipelines/build/common/build_base_file.groovy

```diff
@@ -771,8 +771,8 @@ class Builder implements Serializable {
 
                             if (downstreamJob.getResult() == 'SUCCESS') {
                                 // copy artifacts from build
-                                context.println "[NODE SHIFT] MOVING INTO MASTER NODE..."
-                                context.node("master") {
+                                context.println "[NODE SHIFT] MOVING INTO BUILT-IN NODE..."
```

Comment:
It think this one should be `Controller` to match the other uses.

Reply:
Updated.